src/main/java/com/project/controller/UserController.java

```diff
@@ -25,29 +24,30 @@ public UserController(UserService userService) {
     }
 
     @GetMapping
-    public ResponseEntity<List<User>> getUsers(){
+    public ResponseEntity<List<User>> fetchAllUsers() {
```

Comment:
You don't need to explicitly say that it will fetch `all` users. It is already clear from you `Get` since it has no request parameters.